packages/wdio-cli/src/constants.ts

```diff
@@ -21,7 +21,7 @@ WDIO Configuration Helper
 export const CONFIG_HELPER_SUCCESS_MESSAGE = `
 Configuration file was created successfully!
 To run your tests, execute:
-$ npx wdio run wdio.conf.%s
+$ npx wdio run <path>/wdio.conf.%s
```

Comment:
```suggestion
$ npx wdio run %s/wdio.conf.%s
```

Reply:
Then we need to update the `format` function where this const is used to add the path if conditions are met.